alacritty/src/input.rs

```diff
@@ -791,7 +788,7 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
             };
 
             self.ctx.window_mut().set_mouse_cursor(new_icon);
-        } else if !self.ctx.search_active() {
+        } else {
```

Comment:
If I remember that code correctly, you've enabled certain mouse interactions you shouldn't enable, like Paste on middle click, and since your PR just adds ability to select, you should restrict `PasteSelection`.

Reply:
That's a great point, I'll have to look into that. All we want to allow is just selection I think.